app/pathogen/arbovirus/dashboard/filters.tsx

```diff
@@ -50,7 +50,8 @@ const buildFilterDropdown = (
   placeholder: string,
   state: ArboContextType,
   filterOptions: string[],
-  data: any
+  data: any,
+  { key }: { key: string } // add key prop
```

Comment:
Is this prop valuable? I don't think it's used anywhere. The divs already have keys on them, can we just remove  this?

Reply:
I was getting a warning that there were no keys so i just put this in to get rid of that warning. I can take it out and see if what happens.

Reply:
Yeah appears to not to be valuable as you said the divs have the keys on them